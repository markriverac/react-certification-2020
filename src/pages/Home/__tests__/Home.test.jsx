import React from 'react';
import renderer from 'react-test-renderer';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import NotFoundPage from '../../NotFound/NotFound.page';

Enzyme.configure({ adapter: new Adapter() });

it('Renders without exploding', () => {
  const home = renderer.create(<NotFoundPage />);
  const tree = home.toJSON();
  expect(tree).toMatchSnapshot();
});
