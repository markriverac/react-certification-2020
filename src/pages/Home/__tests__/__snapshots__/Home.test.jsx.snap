// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders without exploding 1`] = `
<section
  className="not-found"
>
  <img
    alt="page not found"
    src="404.gif"
  />
</section>
`;
